examples/ads.amp.html

```diff
@@ -685,17 +686,17 @@ <h2 id="flite">Flite</h2>
     <div fallback></div>
   </amp-ad>
 
-  <h2 id="fusion">Fusion</h2> 
-  <amp-ad width=600 height=100 
+  <h2 id="fusion">Fusion</h2>
+  <amp-ad width=600 height=100
```

Comment:
since you're touching the line, could you help to add double quote around the values?